src/node/tic-tac-toe.js

```diff
@@ -0,0 +1,4 @@
+const Game = require('./game.js');
+
+const ticTacToe = Game();
```

Comment:
Nice! I liked how you already named the game as ticTacToe, so whoever wants to use this project, knows it should start by looking this file. Also, if you parametrize the game by gridSize, you could actually have multiple types of games, like `traditional` `gigantic tictactoe` `medium tictactoe` etc.